src/System.Reflection.Metadata/specs/PE-COFF.md

```diff
@@ -29,10 +29,27 @@ The associated .pdb file may not exist at the path indicated by Path field. If i
 
 If the containing PE/COFF file is deterministic the Guid field above and DateTimeStamp field of the directory entry are  calculated deterministically based solely on the content of the associated .pdb file. Otherwise the value of Guid is random and the value of DateTimeStamp indicates the time and date that the debug data was created.
 
-*Version Major=0x0100, Minor=0x504d* of the data format has the same structure as above. The Age shall be 1. The format of the associated .pdb file is Portable PDB. Together 16B of the Guid concatenated with 4B of the TimeDateStamp field of the entry form a PDB ID that should be used to match the PE/COFF image with the associated PDB (instead of Guid and Age). Matching PDB ID is stored in the #Pdb stream of the .pdb file.
+*Version Major=any, Minor=0x504d* of the data format has the same structure as above. The Age shall be 1. The format of the associated .pdb file is Portable PDB. The Major version specified in the entry indicates the version of the Portable PDB format. Together 16B of the Guid concatenated with 4B of the TimeDateStamp field of the entry form a PDB ID that should be used to match the PE/COFF image with the associated PDB (instead of Guid and Age). Matching PDB ID is stored in the #Pdb stream of the .pdb file.
 
 ### Deterministic Debug Directory Entry (type 16)
 
 The entry doesn't have any data associated with it. All fields of the entry, but Type shall be zero.
 
 Presence of this entry indicates that the containing PE/COFF file is deterministic. 
+
+### Embedded Portable PDB Debug Directory Entry (type 17)
+
+Declares that debugging information is embedded in the PE file at location specified by PointerToRawData. 
+
+*Version Major=any, Minor=0x0100* of the data format:
+
+| Offset | Size           | Field            | Description                                           |
+|:-------|:---------------|:-----------------|-------------------------------------------------------|
+| 0      | 4              | UncompressedSize | The size of decompressed Portable PDB image           |
+| 4      | SizeOfData - 4 | PortablePdbImage | Portable PDB image compressed using Deflate algorithm | 
```

Comment:
Does the Deflate algorithm have some sort of magic number at the start of it? Since I don't know of any good central repository for the IMAGE_DEBUG_TYPE_* constants, it would be good if we had a magic number that we could check to verify this really is an embedded portable PDB and not some custom format that we don't understand.